malos/driver.proto

```diff
@@ -321,6 +321,9 @@ message GpioParams {
 
   // GPIO value
   uint32 value = 3;
+
+  // GPIO vector value
+  uint32 vector = 4;
```

Comment:
"vector" is better for a generic name, not for a field name. "values". Something more appropriate that lets the user understand what the field is for. Specific is better than generic here.